build/yamls/antrea-ipsec.yml

```diff
@@ -717,8 +717,9 @@ data:
     # for the GRE tunnel type.
     enableIPSecTunnel: true
 
-    # CIDR Range for services in cluster. It's required to support egress network policy, should
-    # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
+    # ClusterIP CIDR range for Services. It's required when AntreaProxy is not enabled, and should be
+    # set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver. When
+    # AntreaProxy is enabled, this parameter is not needed.
```

Comment:
is not needed... or is ignored? the former expects that this parameter should not be set.. the latter conveys that even if it was set.. it will be ignored

Reply:
How about "the parameter needs not to be set"?

Reply:
I think just go with "this parameter is not needed and will be ignored if provided"... :)

Reply:
Sure.